packages/server/src/modules/activity/service.ts

```diff
@@ -129,12 +129,19 @@ export const findActivities = (
       });
   });
 
-export const canEditActivities = (
+export const canEditActivities = async (
```

Comment:
I think we won't need this function in the future since our permission infrastructure will be generic enough to support validation.

Reply:
On a side note, this function here presents a specific validation problem when we want to validate permissions for multiple activities.